src/vehicles/components/ScooterSheet.tsx

```diff
@@ -144,27 +160,34 @@ const getRange = (rangeInMeters: number, language: Language) => {
     rangeInMeters > 5000
       ? (rangeInMeters / 1000).toFixed(0)
       : formatDecimalNumber(rangeInMeters / 1000, language, 1);
-  return `ca ${rangeInKm}km`;
+  return `ca ${rangeInKm} km`;
 };
 
 const useSheetStyle = StyleSheet.createThemeHook((theme) => ({
   button: {
     marginTop: theme.spacings.medium,
   },
-  container: {
+  vehicleInfo: {
     flexGrow: 1,
     flexShrink: 0,
+    gapCol: theme.spacings.medium,
```

Comment:
Is this the property that doesn't work yet? If so, should it be removed for now, so it doesn't suddenly start to be behave in unexpected ways?

Reply:
You are right. This is just a left over from a previous iteration. I'll remove it.